content/07.pathogenesis.md

```diff
@@ -96,6 +96,20 @@ While these may be valuable resources, it is important to note that the estimate
 - Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover
 - Cytokines and other soluble factors contribution to immune response
 
+Pathogens infect hosts and are fought by host immune defenses.
+However, pathogens have developed ways to evade the immune system as well as various ways to manipulate the host immunity.
+Other general complexities to consider include the following: disease symptoms can be due to a pathogen or due to the reaction of the immune system to the pathogen; the host immune system can defend against infection but it can also sometimes cause too much damage to the host [@isbn: 9780815332183; @isbn: 9780071283663].
+For SARS-CoV-2, we briefly mention here some of the reviews which describe the primary research literature so far along with some context.
+There is a review on interferon-mediated innate antiviral responses initiated by the virus-infected cells themselves, which can sometimes have inflammatory and other systemic effects [@doi: 10.1016/j.chom.2020.05.008].
+Also, there is a review on the innate immune system response with a focus on innate immune cells and responses other than the interferon-mediated responses; the review also approaches the question of how the innate immune response can confer protection or induce pathogenesis through a dysregulated response [@doi: 10.1016/j.chom.2020.05.009].
+There are also acquired immunity responses to SARS-CoV-2.
+There is a review on the levels of different SARS-CoV-2-specific antibodies detected over the course of an infection [@doi: 10.3389/fimmu.2020.00879].
+A review of T-cell mediated responses to respiratory coronaviruses notes a large reduction in T-cells associated with the acute phase of SARS (from SARS-CoV-1) [@doi: 10.1007/s12026-014-8534-z].
+So far, some articles and reviews on SARS-CoV-2 mention evidence for reduction in the number of CD4 and CD19 T-cells as well as NK-cells [@doi: 10.1016/j.chom.2020.04.009; @doi: 10.1016/j.chom.2020.05.009].
+As a brief immunology background primer for vaccines, which we discuss later, we note that adaptive immunity also has mechanisms for long-term immunity: during a first exposure to a pathogen innate and also later acquired immune responses operate; at the same time, the acquired immune system forms memory B cells and memory T cells based on this experience so that on a possible future second exposure to the same pathogen an earlier/faster and stronger acquired immune response is generated which largely prevents or greatly reduces infection.
+Vaccines are in general attenuated or inactivated pathogens or antigen parts of pathogens which are meant to generate long-term memory of a pathogen or antigens of a pathogen so that if or when a person is exposed to the actual pathogen in the future they will have a strong acquired immunity memory response similar to what occurs upon second exposure to a pathogen; we will describe specific research into vaccines in detail later.
```

Comment:
```suggestion
Vaccines are in general attenuated or inactivated pathogens or antigen parts of pathogens which are meant to generate long-term memory of a pathogen so that if a person is exposed to the same pathogen in the future they will have a strong immunity memory response; we will describe specific research into vaccines in detail later.
```

Reply:
incorporated feedback/edits from Dr. Nadia Danilova.